src/Compilers/CSharp/Portable/Binder/Semantics/Conversions/ConversionsBase.cs

```diff
@@ -3975,6 +3975,26 @@ private bool HasImplicitSpanConversion(TypeSymbol? source, TypeSymbol destinatio
                     return hasCovariantConversion(elementType, spanElementType, ref useSiteInfo);
                 }
             }
+            // SPEC: From `System.Span<Ti>` to `System.ReadOnlySpan<Ui>`, provided that `Ti` is covariance-convertible to `Ui`.
+            // SPEC: From `System.ReadOnlySpan<Ti>` to `System.ReadOnlySpan<Ui>`, provided that `Ti` is covariance-convertible to `Ui`.
+            else if (source.OriginalDefinition.IsSpan() || source.OriginalDefinition.IsReadOnlySpan())
```

Comment:
>OriginalDefinition [](http://example.com/codeflow?start=28&length=18)

Can `.OriginalDefinition` be dropped for the calls to `IsSpan()` and `IsReadOnlySpan()`?